shell/package.json

```diff
@@ -24,8 +24,8 @@
         "moment": "^2.29.1",
         "post-robot": "^10.0.46",
         "prop-types": "^15.7.2",
-        "react": "^16.8.6",
-        "react-dom": "^16.8.6",
+        "react": "^18",
+        "react-dom": "^18",
```

Comment:
I suppose this can't be a union range if we want the test to work...?  We don't really use that test for anything though, so shouldn't be a showstopper *if* we want to make this a union (not sure we do)